website/content/en/docs/building-operators/golang/webhook.md

```diff
@@ -0,0 +1,300 @@
+
+```sh
+Writing kustomize manifests for you to edit...
+Writing scaffold for you to edit...
+api/v1alpha1/memcached_webhook.go
+```
+
+The `--defaulting` flag will scaffold the resources required for a mutating webhook, and the `--programmatic-validation` flag will scaffold the resources required for a validating webhook. 
+In this case we scaffolded both.
+
+After running the `create webhook` command the file structure will change to match the one shown as below.
+
+```sh
+├── Dockerfile
+├── Makefile
+├── PROJECT
+├── api
+│   └── v1alpha1
+│       ├── groupversion_info.go
+│       ├── memcached_types.go
+│       ├── memcached_webhook.go
+│       ├── webhook_suite_test.go
+│       └── zz_generated.deepcopy.go
+├── bin
+│   └── controller-gen
+├── config
+│   ├── certmanager
+│   │   ├── certificate.yaml
+│   │   ├── kustomization.yaml
+│   │   └── kustomizeconfig.yaml
+│   ├── crd
+│   │   ├── kustomization.yaml
+│   │   ├── kustomizeconfig.yaml
+│   │   └── patches
+│   │       ├── cainjection_in_memcacheds.yaml
+│   │       └── webhook_in_memcacheds.yaml
+│   ├── default
+│   │   ├── kustomization.yaml
+│   │   ├── manager_auth_proxy_patch.yaml
+│   │   ├── manager_config_patch.yaml
+│   │   ├── manager_webhook_patch.yaml
+│   │   └── webhookcainjection_patch.yaml
+│   ├── manager
+│   │   ├── controller_manager_config.yaml
+│   │   ├── kustomization.yaml
+│   │   └── manager.yaml
+│   ├── manifests
+│   │   └── kustomization.yaml
+│   ├── prometheus
+│   │   ├── kustomization.yaml
+│   │   └── monitor.yaml
+│   ├── rbac
+│   │   ├── auth_proxy_client_clusterrole.yaml
+│   │   ├── auth_proxy_role.yaml
+│   │   ├── auth_proxy_role_binding.yaml
+│   │   ├── auth_proxy_service.yaml
+│   │   ├── kustomization.yaml
+│   │   ├── leader_election_role.yaml
+│   │   ├── leader_election_role_binding.yaml
+│   │   ├── memcached_editor_role.yaml
+│   │   ├── memcached_viewer_role.yaml
+│   │   ├── role_binding.yaml
+│   │   └── service_account.yaml
+│   ├── samples
+│   │   ├── cache_v1alpha1_memcached.yaml
+│   │   └── kustomization.yaml
+│   ├── scorecard
+│   │   ├── bases
+│   │   │   └── config.yaml
+│   │   ├── kustomization.yaml
+│   │   └── patches
+│   │       ├── basic.config.yaml
+│   │       └── olm.config.yaml
+│   └── webhook
+│       ├── kustomization.yaml
+│       ├── kustomizeconfig.yaml
+│       └── service.yaml
+├── controllers
+│   ├── memcached_controller.go
+│   └── suite_test.go
+├── go.mod
+├── go.sum
+├── hack
+│   └── boilerplate.go.txt
+└── main.go
+
+19 directories, 53 files
+```
+
+The scaffoled file `api/v1alpha1/memcached_webhook.go` has `ValidateCreate`, `ValidateUpdate`, and `ValidateDelete` functions, 
+which allows you to perform different validations based on the operation being performed. The mutating webhook implementation belongs in the `Default` function.
+
+The memcached operator we are building in this example is too simple to require defaulting or additional validations, 
```

Comment:
```suggestion
The example memcached operator explained in the tutorial is too simple to require defaulting or additional validations, 
```
Can you also add the link to tutorial here.

Reply:
Modified.